avatar+badge/avatar.css

```diff
@@ -0,0 +1,135 @@
+@import "../for-import/reset.css";
+@import "../for-import/colors.css";
+
+.avatar-round {
+    border-radius: 50%;
+}
+
+.avatar-square-lg {
+    border-radius: 15px;
+}
+
+.avatar-square-md {
+    border-radius: 10px;
+}
+
+.avatar-square-sm {
+    border-radius: 5px;
+}
+
+.avatar-large {
+    width: 115px;
+    height: 115px;
+}
+
+.avatar-medium {
+    width: 90px;
+    height: 90px;
+}
+
+.avatar-small {
+    width: 65px;
+    height: 65px;    
+}
+
+.avatar-text {
+    background-color: var(--secondary-color);
+    color: var(--default-color);
+    text-align: center;
+    display: inline-block;
+}
+
+.avatar-text-lg {
+    font-size: 2.1rem;
+    padding: 0.5rem;
+    border-radius: 15px;
+}
+
+.avatar-text-md {
+    font-size: 1.6rem;
+    padding: 0.4rem;
+    border-radius: 10px;
+}
+
+.avatar-text-sm {
+    font-size: 1.3rem;
+    padding: 0.3rem;
+    border-radius: 5px;
+}
+
+section {
+    margin: 3rem;
+}
+
+img,
+span {
+    margin-right: 1rem;
+}
+
+
+
+/* Badge Styles */
+.align-horizontal {
+    display: inline-block;
+    margin-right: 1rem;    
+}
+
+.badge-on-avatar-lg {
+    position: relative;
+    width: 120px;    
+}
+
+.icon-top-lg {
+    position: absolute;
+    bottom: 10px;
+    right: 5px;
+    font-size: 2rem;
+    color: var(--primary-color);
+}
+
+.badge-on-avatar-md {
+    position: relative;
+    width: 100px;   
+}
+
+.icon-top-md {
+    position: absolute;
+    bottom: 5px;
+    right: 5px;
+    font-size: 2rem;
+    color:var(--secondary-color);
```

Comment:
**Code Refactor:** As in the classes "icon-top-lg", "icon-top-md" and "icon-top-sm" all the first 4 lines are constant and only the color and bottom properties are changing, so you can make a new class "icon-top" and add the repetitive code in it.